test/Jenkinsfile

```diff
@@ -58,11 +62,27 @@ pipeline {
     )
   }
   stages {
+    stage('clone Setup') {
+      when {
+        expression { env.comment }
+      }
+      steps {
+        sh "git clone https://github.com/ProofOfConceptForJuliSmoothOptimizers/BenchmarkSetup.git || true"
```

Comment:
could we move this repo to JSO?

Reply:
Yes of course. I think it would be wise to have this as a private repo.